spec/rails_helper.rb

```diff
@@ -54,4 +50,11 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  config.before(:suite) do
+    User.destroy_all
+    Bill.destroy_all
+  end
```

Comment:
Oh! Nothing at all, I think I left this comment before I saw yours. I will implement it.